test/functionalTest/cases/3727_avoid_triggering_notification_if_metadata_changes_but_not_the_attribute_value_itself/notifyOnMetadataChange_basic_crud.test

```diff
@@ -0,0 +1,435 @@
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+notfiyOnMetadataChange basic CRUD
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+accumulatorStart --pretty-print
+
+--SHELL--
+
+#
+# 01. POST /v2/subscriptions, to without notifyOnMetadataChange (defaults to true)
+# 02. GET subscription and see notifyOnMetadataChange true
+# 03. PATCH /v2/subscriptions, with notifyOnMetadataChange: false
+# 04. GET subscription and see notifyOnMetadataChange false
+# 05. PATCH /v2/subscriptions, with notifyOnMetadataChange: true
+# 06. GET subscription and see notifyOnMetadataChange true
+# 07. PATCH /v2/subscriptions, with notifyOnMetadataChange: false
+# 08. GET subscription and see notifyOnMetadataChange false
+# 09. PATCH /v2/subscriptions, witout notifyOnMetadataChange (defaults to true)
+# 10. GET subscription and see notifyOnMetadataChange true
+#
+
+
+echo "01. POST /v2/subscriptions, to without notifyOnMetadataChange (defaults to true)"
+echo "================================================================================"
+payload='{
+  "subject": {
+    "entities": [
+      {
+        "id": "E1",
+        "type": "Thing"
+      }
+    ],
+    "condition": {
+      "attrs": [
+        "A1"
+      ]
+    }
+  },
+  "notification": {
+    "http": {
+      "url": "http://localhost:'$LISTENER_PORT'/notify"
+    }
+  }
+}'
+orionCurl --url /v2/subscriptions --payload "$payload"
+subId=$(echo "$_responseHeaders" | grep Location | awk -F/ '{ print $4 }' | tr -d "\r\n")
+echo
+echo
+
+
+echo "02. GET subscription and see notifyOnMetadataChange true"
+echo "========================================================"
+orionCurl --url /v2/subscriptions/$subId
+echo
+echo
+
+
+echo "03. PATCH /v2/subscriptions, with notifyOnMetadataChange: false"
+echo "==============================================================="
+payload='{
+  "subject": {
+    "entities": [
+      {
+        "id": "E1",
+        "type": "Thing"
+      }
+    ],
+    "condition": {
+      "attrs": [
+        "A1"
+      ],
+      "notifyOnMetadataChange": false
+    }
+  }
+}'
+orionCurl --url /v2/subscriptions/$subId -X PATCH --payload "$payload"
+echo
+echo
+
+
+echo "04. GET subscription and see notifyOnMetadataChange false"
+echo "========================================================="
+orionCurl --url /v2/subscriptions/$subId
+echo
+echo
+
+
+echo "05. PATCH /v2/subscriptions, with notifyOnMetadataChange: true"
+echo "=============================================================="
+payload='{
+  "subject": {
+    "entities": [
+      {
+        "id": "E1",
+        "type": "Thing"
+      }
+    ],
+    "condition": {
+      "attrs": [
+        "A1"
+      ],
+      "notifyOnMetadataChange": true
+    }
+  }
+}'
+orionCurl --url /v2/subscriptions/$subId -X PATCH --payload "$payload"
+echo
+echo
+
+
+echo "06. GET subscription and see notifyOnMetadataChange true"
+echo "=============================================================="
```

Comment:
```suggestion
echo "========================================================"
```

Reply:
Fixed in [3ba1ead](https://github.com/telefonicaid/fiware-orion/pull/4310/commits/3ba1ead3128937e80932ee1f3210bacc23f3d475)